build/steps-build-android.yml

```diff
@@ -90,6 +90,7 @@ steps:
     projectDirectory: '${{ parameters.pathToSrc }}/app'
     verboseMode: true
     dartDefine: ENV=$(applicationEnvironment)
+    dartDefine: BUGSEE_TOKEN=$(AndroidDevToken)
```

Comment:
The `AndroidDevToken` name is too broad. It's not clear enough that it's a Bugess token.
(Same comment below for iOSDevToken)